examples/decoding/plot_decoding_spoc_CMC.py

```diff
@@ -0,0 +1,87 @@
+"""
+====================================
+Continuous Target Decoding with SPoC
+====================================
+
+Source Power Comodulation (SPoC) [1] allows to identify the composition of
+orthogonal spatial filters that maximally correlate with a continuous target.
+
+SPoC can be seen as an extension of the CSP for continuous variables.
+
+Here, SPoC is applied to decode the (continuous) fluctuation of an
+electromyogram from MEG beta activity [2].
```

Comment:
This would be [2]_ but it's a website, not an academic reference, so let's just use a ReST link here instead.